server.js

```diff
@@ -12,6 +12,8 @@ app.engine("handlebars", exphbs({ defaultLayout: "main" }));
 app.set("view engine", "handlebars");
 
 app.use("/", require("./routes/htmlroutes"));
+app.use("/user", require("./routes/createuserroute"));
```

Comment:
Gotta make sure you include your routes here in the server file. Basically what this is doing is if it doesn't match a url in the first route file, fall back to this one. You'll also notice I changed the routes in the createuserroute.js file to be just `/`. This means that `localhost:8080/users` will match those route handlers. Otherwise, if it was still `/submit` in that handler, you would have to hit `/users/submit`, which is a little too verbose. Basically you're gonna want most of your basic user handlers to be the same endpoint, the only difference being the request method in order to follow good CRUD practices. For example,
GET /user will show information about a user
POST /user will allow you to create a user
PUT /user will allow you to update a user (change password maybe)
DELETE /user will delete a user